docs/source/deployment/databricks.md

```diff
@@ -259,3 +259,9 @@ Kedro-Viz can then be launched in a new browser tab with the `%run_viz` line mag
 ```ipython
 In [2]: %run_viz
 ```
+
+### How to use Datasets stored on Databricks DBFS
+
+DBFS is a distributed file system mounted into a DataBricks workspace and accessible on a DataBricks cluster. It maps cloud object storage URIs to relative paths so as to simplify the process of persisting files. With DBFS, libraries can read from or write to distributed storage as if it's a local file.
+To use datasets like `SparkDataSet` with DBFS, the file path must be prefixed with `/dbfs`.
+> **Note**: Datasets that use the local filesystem, such as Kedro's Pandas datasets, will return a `DataSetError` if the code tries to load files from a remote DBFS save location. Do not use the `/dbfs` prefix for these datasets.
```

Comment:
Does this also apply if the dataset is stored in the cloud like S3? I'm just a bit confused about how you can know which datasets need `dbfs` and which don't?

Reply:
Only `SparkDataSet` requires the prefix, because of the unique way it resolves its file path. No other data sets work in this way. I have opened a [ticket](https://github.com/kedro-org/kedro-plugins/issues/117) to fix this.